app/routers/sessions.py

```diff
@@ -59,9 +66,15 @@ async def create_session(session: Session):
                         )
                     )
     else:
-        session["is_first"] = False
+        if previous_session["events"] is None:
+            # later, we can avoid creating new session too
+            session["is_first"] = True
+        else:
+            session["is_first"] = False
+
+        session["events"] = previous_session.get("events", None)
         session["has_quiz_ended"] = previous_session.get("has_quiz_ended", False)
-        session["has_quiz_started"] = previous_session.get("has_quiz_started", False)
+        session["time_remaining"] = previous_session.get("time_remaining", None)
```

Comment:
If the user didn't click `start quiz` the very first time and reloaded, we decided on _not_ to create a new session right?

Reply:
ya i thought it could be a separate PR.. see line 70. 

because i was not sure how to handle when `resume` is not clicked.
session1 = start quiz
session2 = reload browser but don't click resume
session3 = reload browser again and click resume

 in session3, `previous_session["events"]` is not None (it has start-quiz event), but nothing happened in session2. so we could as well return session2 only. 

i felt its worthwhile to store info of cases when users just reload browsers without clicking anything, maybe useful for analysis. lets keep creating new sessions, unless we observe its a common pattern that is hurting db?

Reply:
I didn't understood this part, maybe you can explain on a quick call?